ui/src/utils/cluster.utils.ts

```diff
@@ -19,10 +19,9 @@ export function generateClusterName(username = ''): string {
   // prepare to combine, but filter any empty part (only one that should ever be empty is user string)
   const nameArray = [userPart, datePart, randomPart].filter(Boolean);
 
-  // combine the 3 parts, truncate it at 28 characters, and remove a trailing '-', if any.
-  // Truncation is needed to allow generating wildcard certificates for OpenShift using Let's Encrypt: since LE insists
-  // on the domain forming the Common Name of the certificate, the string '*.apps.<name>.openshift.infra.rox.systems'
-  // must not exceed 64 characters. This leaves a budget of 29 characters for the <name> portion.
-  // RS-171 - 29 chars raises openshift error: "openshift_public_hostname must be 63 characters or less"
-  return nameArray.join('-').slice(0, 28).replace(/-$/, '');
+  // ROX-15492
+  //   - OCP 3.11 allows us a maximum of up to 27 characters to fulfil: "openshift_public_hostname must be 63 characters or less".
+  //   - OCP 4 demo: Let's Encrypt allows common names of up to 63 characters. With the format '*.apps.<name>.openshift.infra.rox.systems', 27 characters remain for the name.
```

Comment:
`*.apps.<name>.openshift.infra.rox.systems` are 36 chars (exluding `<name>`):

```
$ echo "*.apps..openshift.infra.rox.systems"  | wc
       1       1      36
```

Let's Encrypt allows a maximum of up to including 63 chars for CN: https://letsencrypt.org/docs/glossary/#:~:text=For%20roots%20and%20intermediates%20it's,is%20limited%20to%2063%20characters.

That leaves 27 chars.